tft.py

```diff
@@ -792,6 +780,27 @@ def get_newer_version() -> tuple[str, str] | None:
     return None
 
 
+def re_add_non_debug_logger(log_level: str) -> None:
+    """
+    Remove all loggers and add our formatted logger at a desired level.
+    Since loguru is not loggers in a strict sense but more about log handlers that do not care about the level,
+    there is no straight setLevel method. So to change the level, we have to re-add it.

```

Comment:
Dang, that's quite annoying! I find that super weird we need to re-add it with a lesser debug level 🤔

Reply:
Yeah I was surprised as well, but understand the reasoning. It's also really only a requirement because I felt it would make sense to ignore the log level for our start up messages.
Now that I think about it, we could also just add a second logger sink to stderr and remove it after startup, that would also work.

Reply:
Yeah, I think this flow is good enough, and glad you documented that in the code so it should help avoid future questions (and could help someone think about a workaround / different way to do it) 👌